pipeline/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -714,6 +725,64 @@
     "        ${(\"--stranded \" + stranded) if stranded else \"\"}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5dd533f4-c308-4b2c-a3f9-dc54e258e82c",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "The RNASEQC results were merged in the following step"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a2b8cf8-b9a1-4440-a86f-3fbbb610e61b",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[rnaseqc_call_5]\n",
```

Comment:
Do we need to use the aggregate rnaseqc result script from Broad?

Reply:
No we dont need to, but I thought it was preferred to use whatever they have for the pipeline?

Reply:
Yes, it's preferred to use their code